komodo/build.py

```diff
@@ -56,9 +56,10 @@ def cmake(pkg, ver, path, prefix, builddir,
              '-DBUILD_SHARED_LIBS=ON',
              '-DCMAKE_PREFIX_PATH={}'.format(fakeprefix),
              '-DCMAKE_MODULE_PATH={}/share/cmake/Modules'.format(fakeprefix),
-             '-DCMAKE_INSTALL_PREFIX={}'.format(prefix)
+             '-DCMAKE_INSTALL_PREFIX={}'.format(prefix),
+             '-DDEST_PREFIX={}'.format(fakeroot)
```

Comment:
Can you add a trailing `,` and a comment `  #  hack, see also Statoil/sunbeam#89`  ?

The `DDEST_PREFIX` will create a warning in all non-sunbeam cmake projects, right?

Reply:
OK - I have added more comments describing the issue, both here and in particular here: https://github.com/Statoil/sunbeam/pull/89.

Have not added a trailing `,` - I really don't like that technique.

Reply:
Fair enough.  The trailing comma is just so that the changes (diff) only touch the actually changed lines.  But nevermind.  Approved!

Reply:
Good - thank you; I agree the technique is "diff friendly" - but when actually reading the code I think it looks ugly. Anyway - in the mind of the beholder.